pkg/controller/volume/attachdetach/statusupdater/node_status_updater.go

```diff
@@ -66,7 +66,7 @@ func (nsu *nodeStatusUpdater) UpdateNodeStatuses() error {
 			// If node does not exist, its status cannot be updated.
 			// Do nothing so that there is no retry until node is created.
 			klog.V(2).Infof(
-				"Could not update node status. Failed to find node %q in NodeInformer cache. Error: '%v'",
```

Comment:
Can you please rephrase so it works without `Error:`? Something like "Could not update node status: %s". `err` usually contains "node XYZ not found" or something like that. IMO we don't need to mention NodeInformer - almost everything has informer in the controller.

In addition, fixing all "Error:" in the attach/detach controller would be great!

Reply:
Ok,I will fix these later.